modules/waf-regional/wafregional_ruleset10_blacklisted_ips.tf

```diff
@@ -14,33 +14,14 @@ resource "aws_wafregional_rule" "detect_blacklisted_ips" {
   }
 }
 
-resource "aws_wafregional_ipset" "blacklisted_ips" {
+resource aws_wafregional_ipset blacklisted_ips {
   name = "${var.waf_prefix}-generic-match-blacklisted-ips"
-  dynamic "ip_set_descriptor" {
+  dynamic ip_set_descriptor {
     for_each = var.blacklisted_ips
-    content {
-      //resource "aws_waf_ipset" "test" {
-      //  name = "test"
-      //
-      //  dynamic "ip_set_descriptors" {
-      //    # The for_each argument is a hardcoded list in this illustrative example,
-      //    # however it can be sourced from a variable or local value as well as
-      //    # support multiple argument values as a map.
-      //    for_each = ["1.1.1.1/32", "2.2.2.2/32", "3.3.3.3/32"]
-      //
-      //    content {
-      //      type  = "IPV4"
-      //      value = ip_set_descriptors.value
-      //    }
-      //  }
-      //}
-      //
-      //output "test" {
-      //  value = aws_waf_ipset.test.ip_set_descriptors[*].value
-      //}
 
+    content {
       type  = "IPV4"
-      value = ip_set_descriptors.value
+      value = ip_set_descriptor.value
```

Comment:
:handshake:

Reply:
@lucasvinals Many thanks for your collab here :100:

Reply:
Glad it helps 😄